prog/weaver/http.go

```diff
@@ -118,6 +111,26 @@ var rootTemplate = template.New("root").Funcs(map[string]interface{}{
 	"trimSuffix": strings.TrimSuffix,
 })
 
+func countDNSEntries(entries []nameserver.EntryStatus) int {
```

Comment:
The design for `status` was to put just data-collecting spread around the various modules and to put all summarising and aggregation in `http.go`.  We could change that, but should probably discuss it with @awh